Sources/RichTextKit/RichTextView_UIKit.swift

```diff
@@ -183,6 +183,20 @@ open class RichTextView: UITextView, RichTextViewComponent {
 
     // MARK: - Open Functionality
 
+    open func renderLinks(in characterRange: NSRange, at origin: CGPoint) {
```

Comment:
These two are almost identical, but should be able to be identical as well, right? If so, can we instead extend the component protocol?